website/docs/tutorials/07-testing/02-cheatcodes/skip.md

```diff
@@ -0,0 +1,31 @@
+# `skip`
+```python
+def skip(reason: Optional[str] = None) -> None:
+```
+Skip a test.
+
+:::warning
+This cheatcode is only available in test's setup.
+:::
+
+```cairo
+%lang starknet
+
+@external
+func setup_function() {
+    %{ skip("Reason") %} // <-- ok
+    return ();
+}
+
+@external
```

Comment:
You have already said that in the warning block. No need to repeat info in example :)

Reply:
To be honest I'm not sure, it's easier to find it here.

Reply:
I don't share this opinion. Would love to hear from @MaksymilianDemitraszek what he thinks about this.

Reply:
I agree with @mkaput , it is a bit wordy to include 2/3 of example code just to show it doesn't work like that

Reply:
It is nice you tried explain it carefully, but it is generally nice when we keep [explaining in details measure] even in whole docs, and we don't provide such examples for other cheatcodes

Reply:
Done